tsconfig.json

```diff
@@ -14,14 +15,18 @@
     "esModuleInterop": true,
     "noImplicitOverride": true,
     "noImplicitAny": false,
-    "outFile": "dist/types/index.d.ts"
+    "outFile": "dist/types/index.d.ts",
+    "module": "NodeNext",
```

Comment:
this solves the current build failure

Reply:
Can you elaborate? Why does setting the `module` option fix the VIPER build?

Reply:
"This works but I don't think this is necessary"

Reply:
On further investigation, looks like there were 2 separate issues.

1. `tsc` by default targets `ES5`, which doesn't support the `#private` syntax.
2. `module` by default is `CommonJS` if we're using `ES5` which, since it's not specified, the `target` option defaults to.

Then something about viper not having its own tsconfig causes even more build inconsistency and weirdness.

Reply:
I've updated the PR body with my findings